net/quic/quic_chromium_packet_reader.cc

```diff
@@ -39,7 +43,126 @@ QuicChromiumPacketReader::QuicChromiumPacketReader(
 
 QuicChromiumPacketReader::~QuicChromiumPacketReader() = default;
 
+#if defined(STARBOARD)
+int QuicChromiumPacketReader::StartReadingMultiplePackets() {
+  for (;;) {
```

Comment:
Can this ever busyloop and not break the loop here ? should there be a loop limit / counter ?

Reply:
This mirrors the loop in `StartReading` (lines 147+), the implementation that reads one packet at a time.

There is a limit triggering breaks, both count and clock based with `yield_after_packets_` and `yield_after_`.

Additionally, the sockets are always nonblocking, so as soon as there is no data `ERR_IO_PENDING` will be returned and the function returns (the `OnReadMultiplePacketComplete` is called from the socket watcher when there is data available to continue reading).

Reply:
Ack - thanks.